bdd/spec/027_seq_spec.sh

```diff
@@ -160,15 +160,15 @@ error: sequence out-of-bounds"
             if i < 2 {
               return ok(1);
             } else {
-              const prev = self.recurse(i - 1);
-              const prevPrev = self.recurse(i - 2);
+              const prev = self.recurse(i - 1 || 0);
+              const prevPrev = self.recurse(i - 2 || 0);
               if prev.isErr() {
                 return prev;
               }
               if prevPrev.isErr() {
                 return prevPrev;
               }
-              return ok((prev || 1) + (prevPrev || 1));
+              return (prev || 0) + (prevPrev || 0);
```

Comment:
Oh, because I was silly here. Let me remove that.

Reply:
Wait, no. I think there's an actual issue here wrt the inferred type of the variable because of how `recurse` has that opaque `Self` type with no generics data to help, and that's why I put that in there.

I will update with a TODO on when recurse has proper type inference.